Refit-Tests/MultipartTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Reflection;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Refit;
+
+namespace Refit.Tests
+{
+    public interface IRunscopeApi
+    {
+        [Multipart]
+        [Post("/")]
+        Task<HttpResponseMessage> UploadStream([AttachmentName("test.pdf")] Stream stream);
+
+        [Multipart]
+        [Post("/")]
+        Task<HttpResponseMessage> UploadBytes([AttachmentName("test.pdf")] byte[] bytes);
+
+        [Multipart]
+        [Post("/")]
+        Task<HttpResponseMessage> UploadString(string someString);
+
+        [Multipart]
+        [Post("/")]
+        Task<HttpResponseMessage> UploadFileInfo(FileInfo fileInfo);
+    }
+
+    [TestFixture]
+    public class MultipartTests
```

Comment:
Can you [Skip] these tests? I still want them in here because they're useful but having them all fail by default is :crying_cat_face: